manual/src/SUMMARY.md

```diff
@@ -31,3 +31,4 @@
   - [btc](examples/btc/examples.md)
     - [todo](examples/btc/todo.md)
   - [Metrics](examples/metrics/metrics.md)
+  - [sftpgo-sal](examples/sftpgo/sal.md)
```

Comment:
Please update here so the examples are in a list (so they show up in the side navigation)